README.rst

```diff
@@ -58,9 +58,10 @@ generators, there is broad support.
 Apart from that, jedi-vim supports the following commands
 
 - Completion ``<C-Space>``
-- Goto assignments ``<leader>g`` (typical goto function)
-- Goto definitions ``<leader>d`` (follow identifier as far as possible,
+- Goto assignment ``<leader>g`` (typical goto function)
+- Goto definition ``<leader>d`` (follow identifier as far as possible,
   includes imports and statements)
+- Goto stub ``<leader>s``
```

Comment:
```suggestion
- Goto (typing) stub ``<leader>s``
```
?

Reply:
and/or add an explanation similar to the lines above.

Reply:
I personally find those explanations kind of redundant.